src/renderer/views/Settings/Settings.vue

```diff
@@ -1,34 +1,49 @@
 <template>
-  <div>
+  <div class="settings settingsPage">
     <template v-if="unlocked">
-      <div class="switchRow">
-        <ft-toggle-switch
-          class="settingsToggle"
-          :label="$t('Settings.Expand All Settings Sections')"
-          :default-value="allSettingsSectionsExpandedByDefault"
-          :compact="false"
-          @change="updateAllSettingsSectionsExpandedByDefault"
-        />
-        <ft-toggle-switch
-          class="settingsToggle"
-          :label="$t('Settings.Sort Settings Sections (A-Z)')"
-          :default-value="settingsSectionSortEnabled"
-          :compact="false"
-          @change="updateSettingsSectionSortEnabled"
+      <div v-show="settingsSectionTypeOpenInMobile != null">
+        <font-awesome-icon
+          :icon="['fas', 'angle-left']"
+          class="returnToMenuMobileIcon"
+          role="button"
+          :title="$t('Settings.Return to Settings Menu')"
+          tabindex="0"
+          @click="settingsSectionTypeOpenInMobile = null"
+          @keydown.space.enter="settingsSectionTypeOpenInMobile = null"
         />
       </div>
-      <template
-        v-for="(settingsComponent, i) in settingsSectionComponents"
+      <ft-settings-menu
+        v-show="isInDesktopView || settingsSectionTypeOpenInMobile == null"
+        :settings-sections="settingsSectionComponents"
+        @navigate-to-section="navigateToSection"
+      />
+      <div
+        v-show="isInDesktopView || settingsSectionTypeOpenInMobile != null"
+        class="settingsContent"
       >
-        <hr
-          v-if="i !== 0"
-          :key="settingsComponent.type + 'hr'"
-        >
-        <component
-          :is="settingsComponent.type"
-          :key="settingsComponent.type + 'component'"
-        />
-      </template>
+        <div class="switchRow">
+          <ft-toggle-switch
+            class="settingsToggle"
+            :label="$t('Settings.Sort Settings Sections (A-Z)')"
+            :default-value="settingsSectionSortEnabled"
+            :compact="false"
+            @change="updateSettingsSectionSortEnabled"
+          />
+        </div>
+        <div class="settingsSections">
+          <template
+            v-for="(settingsComponent) in settingsSectionComponents"
+          >
+            <component
+              :is="settingsComponent.type"
+              :ref="settingsComponent.type"
+              :key="settingsComponent.type + '-component'"
```

Comment:
Now that we no longer have the `<hr>` element, we can just used the type as the key directly.
```suggestion
              :key="settingsComponent.type"
```